interview-questions.md

```diff
@@ -8,62 +8,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: A variable that holds data
 
-  Researched answer:
+  Researched answer: Values that are passed into a function.
 
 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer:
+  Your answer:element is required.
 
-  Researched answer:
+  Researched answer:element, index, array are optional.
 
 
 
 3. What is object destructuring?
 
-  Your answer:
+  Your answer:Taking values from an object??
 
-  Researched answer:
+  Researched answer: JavaScript expression that makes it possible to unpack values from arrays or properties of objects.
 
 
 
 4. What is the difference between a function and a method?
 
-  Your answer:
+  Your answer: A piece of code that performs a task when called.
 
-  Researched answer:
+  Researched answer: A code block designed to perform a task when invoked.
```

Comment:
Again, here the question is asking for the difference between two things. I don't feel like I got the answer I was looking for.